gateway-ha/src/main/java/io/trino/gateway/ha/security/LbAuthorizer.java

```diff
@@ -19,17 +19,17 @@ public boolean authorize(LbPrincipal principal, String role) {
       case "ADMIN":
         log.info("User {} identified as ADMIN", principal.getName());
         return principal.getMemberOf()
-            .filter(m -> m.contains(configuration.getAdmin()))
+            .filter(m -> m.matches(configuration.getAdmin()))
```

Comment:
There should be a test case for this functionality

Reply:
I added test code. PTAL